packages/theme/src/cli/utilities/theme-command.ts

```diff
@@ -51,15 +51,15 @@ export default abstract class ThemeCommand extends Command {
     const {flags} = await this.parse(klass)
 
     // Single environment
-    if (!Array.isArray(flags.environment)) {
+    if (!flags.environment) {
       const session = await this.ensureAuthenticated(flags)
       await this.command(flags, session)
       return
     }
 
     // Synchronously authenticate all environments
     const sessions: {[storeFqdn: string]: AdminSession} = {}
-    const environments = flags.environment
+    const environments = Array.isArray(flags.environment) ? flags.environment : [flags.environment]
```

Comment:
I'd add a comment sharing the oclif parsing context here, to support future readers :)